python/dlr/api.py

```diff
@@ -43,6 +43,15 @@ def _find_model_file(model_path, ext):
         return model_files[0]
     return None
 
+def _find_saved_model(model_path):
+    if not os.path.isdir(model_path):
+        return False
+    model_file = glob.glob(os.path.abspath(os.path.join(model_path, '*' + '.pb')))
```

Comment:
TF2 tf.saved_model.load looks for `saved_model.pbtxt` or `saved_model.pb`.

Reply:
done